core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -355,7 +355,8 @@ class Router(formatOps: FormatOps) {
           case _ => NoSplit
```

Comment:
I will create another PR when it gets fixed. This logic "leftOwner.parent.exists(_.is[Mod])" solves the issue #618.